types/src/validator_verifier.rs

```diff
@@ -339,6 +391,25 @@ impl From<&ValidatorVerifier> for ValidatorSet {
     }
 }
 
+/// Helper function to generate LedgerInfoWithSignature from a set of validator signers used for testing
+pub fn generate_validator_verifier(validators: &[ValidatorSigner]) -> ValidatorVerifier {
```

Comment:
probably add cfg(test)?

Reply:
This is used in a bunch of functions in `test_utils`, which are not guarded by  `cfg(test)`.

Reply:
most test_utils should be cfg(any(test, fuzzing/testing)) though

Reply:
Unfortunately `proptest_type` is not guarded by `cfg(test)`. We can follow up on this to see if we can guard them by test.

Reply:
`proptest_type` is guarded by `#[cfg(any(test, feature = "fuzzing"))]`